src/js-marketing-notification.ts

```diff
@@ -89,11 +89,8 @@ export class JsMarketingNotification {
     const expansionClass = 'marketing-notification--expanded';
     node.classList.toggle(expansionClass);
 
-    if (node.classList.contains(expansionClass)) {
-      if (this.openCallback) this.openCallback(node);
-    }
-    else {
-      if (this.closeCallback) this.closeCallback(node);
-    }
+    if (node.classList.contains(expansionClass) && this.openCallback) return this.openCallback(node);
+
+    if (this.closeCallback) return this.closeCallback(node);
```

Comment:
With above comment, put this in the else and just simply call `this.closeCallback(node);`